src/python/src/README.rst

```diff
@@ -0,0 +1,27 @@
+gRPC Python
+===========
+
+Package for GRPC Python.
+
+Dependencies
+------------
+
+Ensure that you have installed GRPC core.
+
+Either install from source:
+
+::
+
+  git clone https://github.com/grpc/grpc.git
+  cd grpc
+  ./configure
+  make && make install
+
+Or install from a debian package:
```

Comment:
Change the order, e.g.
- on linux, you can install using the deb package
...
- otherwise, you can install from source
...